packages/client/src/ActionsAndReducers/playerUi/playerUi_Reducer.js

```diff
@@ -143,8 +158,7 @@ export const playerUiReducer = (state = initialState, action) => {
       newState.gameDescription = action.payload.data.overview.gameDescription
       newState.allChannels = action.payload.data.channels.channels
       newState.allForces = action.payload.data.forces.forces
-      newState.allPlatformTypes = modifyPlatformTypesBasedOnMessages(action.payload.data.platform_types.platformTypes, newState)
```

Comment:
Aah, @lilitkarapetyan - this is where the `modifyPlatformTypesBasedOnMessages` was previously called. I guess you can delete it now, right?